src/content/docs/infrastructure/prometheus-integrations/integrations-list/install-apache-flink.mdx

```diff
@@ -1,10 +1,12 @@
 ---
-title: Get Apache Flink dashboards in New Relic
+title: Apache Flink integration
 tags:
     - Integrations
     - Prometheus integrations
     - Configure Prometheus OpenMetrics for Flink
```

Comment:
can add the below too. Please cross check if my sentence formation can be corrected
```
    - Apache Flink metrics integration on New Relic platform
```